docs/en/advanced/Command-Line-Interface.md

```diff
@@ -29,6 +11,253 @@ priority: 0
 Alluxio's command line interface provides users with basic file system operations. You can invoke
 the following command line utility to get all the subcommands:
 
+```bash
+$ ./bin/alluxio
+Usage: alluxio [COMMAND]
+       [format [-s]]
+       [getConf [key]]
+       [logLevel]
+       [runTests]
+       ...
+```
+
+## General operations
+
+In this section, we list the usages and examples of general Alluxio operations except file system 
+and file system admin operations. File system operations will be listed in the next section 
+and file system admin operations are listed in the [Admin CLI doc](Admin-CLI.html).
+
+### format
+
+The `format` command will format the Alluxio master and all workers.
+
+If `-s` specified, only format if underfs is local and doesn't already exist.
+
+Note that `format` is only required when you run Alluxio for the first time. 
+If you run this command for an existing Alluxio cluster, all previously stored data 
+and metadata in Alluxio filesystem will be erased. However, data in under storage will not be changed.
+
+```bash
+$ ./bin/alluxio format
+$ ./bin/alluxio format -s
+```
+
+### formatMaster
+
+The `formatMaster` command will format the Alluxio master and all workers.
+
+```bash
+$ ./bin/alluxio formatMaster
+```
+
+### formatWorker 
+
+The `formatWorker` command will format the Alluxio master and all workers.
+
+```bash
+$ ./bin/alluxio formatWorker
+```
+
+### bootstrapConf
+
+The `bootstrapConf` command will generate a configuration file if one doesn't exist.
```

Comment:
can we expand more on how this is meant to be used?

Reply:
DONE, thanks